server/p2p/p2p.go

```diff
@@ -65,7 +65,7 @@ var (
 
 	// userAgentVersion is the user agent version and is used to help
 	// identify ourselves to other kaspa peers.
-	userAgentVersion = fmt.Sprintf("%d.%d.%d", version.AppMajor, version.AppMinor, version.AppPatch)
+	userAgentVersion = version.Version()
```

Comment:
Please use this var in DefaultUserAgent

Reply:
No can do :/
DefaultUserAgent is in wire package.

p2p package extends the user agent with UserAgentComment, so needs to re-do some work.

I guess btcd had DefaultUserAgent in wire because wire was considered a "library package" that should be used outside of btcd.

In our case it's less relevant, and it's only used for tests. I'm not sure it's a problem, but if we decide it is, I think it's better to get rid of it in separate ticket.